packages/launcher/src/renderer/launcher/apps/AppsView.js

```diff
@@ -148,24 +265,56 @@ export default createFragmentContainer(AppsView, {
   apps: graphql`
     fragment AppsView_apps on AppsQuery {
       installed {
-        id
         appID
         name
+        manifest {
+          permissions {
```

Comment:
No need to change this now but as discussed using an unique fragment with the `@relay(mask: false)` directive could be useful for permissions.